doc/sources/license.rst

```diff
@@ -0,0 +1,23 @@
+.. ******************************************************************************
+.. * Copyright 2021 Intel Corporation
```

Comment:
for new files this should be current year

Reply:
Fixed in https://github.com/intel/scikit-learn-intelex/pull/1323/commits/18c610980943defa287851ff154401a7f6fc9911